FetchMigration/python/migration_monitor.py

```diff
@@ -58,24 +63,30 @@ def run(args: MigrationMonitorParams, wait_seconds: int = 30) -> None:
     # TODO Remove hardcoded EndpointInfo
     default_auth = ('admin', 'admin')
     endpoint = EndpointInfo(args.dp_endpoint, default_auth, False)
+    target_doc_count: int = args.target_count
     prev_no_partitions_count = 0
     terminal = False
+    logging.info("Starting migration monitor until target doc count: " + str(target_doc_count))
     while not terminal:
+        time.sleep(wait_seconds)
```

Comment:
Consider renaming this to be more descriptive for the user.  e.g. poll_interval_seconds